Lib/multiprocessing/shared_memory.py

```diff
@@ -464,11 +583,13 @@ def _offset_packing_formats(self):
     def _offset_back_transform_codes(self):
         return self._offset_packing_formats + self._list_len * 8
 
-    @classmethod
-    def copy(cls, self):
+    def copy(self, *, name=None):
```

Comment:
What's the use case of `name`? Copying another `ShareableList` instance? What's the point?

Reply:
When creating `SharedMemory` instances, the user can control the name of the shared memory block (to support a variety of use cases).  Here too, when creating a copy, the user can specify the name of the new shared memory block about to be created to hold the copy.

Reply:
IMO this is an unnecessary over-complication of the API. If the user wants [a copy of] another memory region they can do so by instantiating a new `SharedMemory` instance (which really is what you're doing here). I recommend mimicking the `list` object as much as possible (aka `list.copy` doesn't have a "name" parameter).

Reply:
Agreed -- removed in commit 715ded90abdde274159b219172a844a3a7da8424.